content/news/XXXX-XX-XX-mixxx-2-3-0-release.md

```diff
@@ -0,0 +1,126 @@
+title: Mixxx 2.3.0 released
+authors: Jan Holthuis
+tags: 2.3, 2.3.0, release announcement
+status: draft
+comments: yes
+
+After a long waiting time, we are proud to announce the release of Mixxx 2.3.0, [download it!]({filename}/pages/download.md#stable)
+This release brings [hotcue colors & labels]({filename}/news/2020-08-25-new-in-2-3-hotcue-colors.md), import of [Serato](2021-02-08-new-in-2-3-serato-support.md) and [Rekordbox]({filename}/news/2020-07-20-new-in-2-3-rekordbox-support.md) libraries, [more accurate key detection]({filename}/news/2021-04-08-new-in-2-3-keyfinder.md), [intro/outro sections](2020-07-09-intro-outro-sections.md), a revamped LateNight skin, multithreaded analysis, [deck cloning]({filename}/news/2020-11-15-new-in-2-3-deck-clone.md), [track color support]({filename}/news/2020-10-24-new-in-2-3-track-colors.md) and more.
```

Comment:
we could link to all the bugs fixed in 2.3: 

https://bugs.launchpad.net/mixxx/+bugs?field.searchtext=&orderby=-importance&field.status%3Alist=FIXCOMMITTED&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.milestone%3Alist=83958&field.tag=&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search

If we migrate to GH Issues in time, linking to those would also be an option.

Reply:
It also shows all the bugs which didn't get fixed so I didn't want to link that...

Reply:
We will remove the unfixed bugs from the mile stone. There will be overlapping, though.

Reply:
the release announcement should list new, fancy feature and a few important bug fixes only.
all fixed bugs and small tweaks shouldn't be on the front page IMO

Reply:
Okay, instead of pasting the full changelog I'll just link to it and add the Launchpad link.

Reply:
If this was a list it would be easier to read, and more impressive -- like the [2.2 announcement](https://mixxx.org/news/2018-12-24-mixxx-2-2-released/), but not that detailed probably, or at least we don't need the categories if there are only one or two features in some of them.

Let's collect some 'minor' features / additions, like (browsing [2.3 Done](https://github.com/mixxxdj/mixxx/projects/2#column-8315970) right now)
- Opus encoder added
- NN new controller mappings added
- Preferences and controller GUI polished
- ...

Reply:
maybe also a short 'Behind the scenes' paragraph with all elaborate changes invisible to most users, like
- library improvements
- switch to cmake
- ... ?

Reply:
@ronso0 Intially I planned to copy the changelog from here: https://github.com/mixxxdj/mixxx/blob/1628fc7cb9c27dc1413568e368c182b32f42aa85/CHANGELOG.md#230-unreleased

It doesn't list each small bugfix either (too many). I don't care if we copy it or link to it.

Reply:
I think we can link to that.
and for the announcement itslef we can use a stripped-down version IMO, which contains stuff that most users may care about.
If we're not _toooo_ much in a hurry I can work on that over the weekend (off the grid, and afk actually.. ; )

Reply:
Done.